mythril/laser/ethereum/state/constraints.py

```diff
@@ -30,6 +30,12 @@ def __init__(
         self._default_timeout = 100
         self._is_possible = is_possible
 
+    def compress(self):
+        constraint = True
+        for c in self[:]:
+            constraint = And(constraint, c)
+        return constraint
```

Comment:
```suggestion
        return reduce(And, constraint, True)
```
I think this would work as well (I love python one liners 😅 )
reduce would need to be imported from functools though 😄